### DISCUSSION TITLE
Selection chats where image is attachment

### DISCUSSION THREAD

**Discussion by patrickdalla** [sepinf-inc/IPED#2324] [General]
I've found some relevant images. They are attachments of chats, corretly referenced by IPED. How is the easiest way to select all those chats where this images appear?

**Comment by lfcnassif** [comment]
Copy the hashes of all images from the Metadata Filter panel. Clear all filters. Go to the search bar and search for:
linkedHashes:(hash1 hash2 hash3)

Chats referencing those images should be returned.

**Comment by aberenguel** [comment]
The PR #2288  implements a feature to select the chats using context menu in items table. It is not merged yet, but it is working. I need to review it to check if there are some possible optimizations.

**Comment by lfcnassif** [comment]
@patrickdalla would you have available time to review @aberenguel's PR #2288 implementing this?
